docs/pages/ESSupport.md

```diff
@@ -854,3 +854,45 @@ This analyzer checks every JavaScript file for occurences of a `sap/ui/core/Core
+  Compiles API index resources from all `api.json` resources available in the given test resources directory.
+  The resulting index resources (e.g. `api-index.json`,  `api-index-deprecated.json`,
+  `api-index-experimental.json` and `api-index-since.json`) are mainly to be used in a UI5 SDK.
+
+#### JSDoc: UI5 SDK
+
+a UI5 SDK can be build locally. To get more insights about the local UI5 SDK build setup, have a look at the [Developer Guide](https://github.com/SAP/openui5/blob/master/docs/developing.md#building-the-openui5-sdk-demo-kit).
+
+Currently, the resources needed for a UI5 SDK build are stored in [openui5](https://github.com/SAP/openui5/tree/master/lib/jsdoc) and in [ui5-builder](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc). This is needed because these files are not part of the `sap.ui.core` library artefact, so building JSDoc for any library has no access to the needed resources. Therefore it is necessary to have these resources also available in the *ui5-builder*. This might chance in future.
+
+Following artefacts are part of a UI5 SDK build:
+
+- [createIndexFiles.cjs](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/lib/createIndexFiles.cjs): Creates several index files by interpreting the `sap-ui-version.json`.
+- [transformApiJson.cjs](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/lib/transformApiJson.cjs): Preprocesses `api.json` files for use in a UI5 SDKs. Transforms the `api.json` as created by the JSDoc build into a pre-processed `api.json` file suitable for the SDK. The pre-processing includes formatting of type references, rewriting of links and other time consuming calculations.
+- [plugin.cjs](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/lib/ui5/plugin.cjs): UI5 plugin for JSDoc3. The plugin adds the following UI5 specific tag definitions to JSDoc3.
+
+    - disclaimer
+    - experimental
+    - final
+    - interface
+    - implements
+
+    It furthermore listens to the following JSDoc3 events to implement additional functionality
+
+    - parseBegin: to create short names for all file that are to be parsed
+    - fileBegin: to write some line to the log (kind of a progress indicator)
+    - jsdocCommentFound: to pre-process comments, empty lines are used as paragraph markers a default visibility is added, legacy tag combinations used in JSdoc2 are converted to JSDoc3 conventions
+    - newDoclet
+    - parseComplete: remove undocumented/ignored/private doclets or duplicate doclets
+
+    Last but not least, it implements an astNodeVisitor to detect UI5 specific "extend" calls and to create documentation for the properties, aggregations etc. that are created with the "extend" call.
+
+- [publish.cjs](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/lib/ui5/template/publish.cjs): JSDoc3 template for UI5 documentation generation.
```

Comment:
This is also worth a more detailed description, e.g.

 - adds missing namespaces
 - determines the exports names of exported APIs
 - writes out the api.json from the collected JSDoc information
 - ...